packages/@magic-sdk/types/src/modules/auth-types.ts

```diff
@@ -162,6 +183,17 @@ export type LoginWithMagicLinkEventHandlers = {
   [LoginWithMagicLinkEventEmit.Retry]: () => void;
 } & DeviceVerificationEventHandlers;
 
+export type LoginWithSmsOTPEventHandlers = {
+  // Event sent
+  [LoginWithSmsOTPEventEmit.VerifySmsOtp]: (otp: string) => void;
+  [LoginWithSmsOTPEventEmit.Cancel]: () => void;
+
+  // Event received
+  [LoginWithSmsOTPEventOnReceived.SmsOTPSent]: () => void;
+  [LoginWithSmsOTPEventOnReceived.InvalidSmsOtp]: () => void;
+  [LoginWithSmsOTPEventOnReceived.ExpiredSmsOtp]: () => void;
+};
```

Comment:
Could we add Device verification? Since we have it for SMS
```suggestion
} & DeviceVerificationEventHandlers
```

To test device verification on SMS, you'll need to enable that in the dashboard first. 

login on a desktop browser, then login via your phone shall trigger a device verification flow. One caveat is that, the first login attempt of your account will not trigger the device verification.